crates/rpc/src/types/request.rs

```diff
@@ -7,3 +7,10 @@ pub struct ValidatorsPostRequest {
     pub ids: Option<Vec<ValidatorID>>,
     pub status: Option<Vec<String>>,
 }
+
+#[derive(Debug, Deserialize)]
+pub struct SyncCommitteeSubscription {
+    pub validator_index: String,
+    pub sync_committee_indices: Vec<String>,
+    pub until_epoch: String,
+}
```

Comment:
we don't want to use strings here, we want to use serde with to serialize it properly